### ISSUE TITLE
Add Proper alt Attribute to All Images  

### ISSUE THREAD

**Issue by vkoves** [vkoves/vkoves.github.io#9]
HTML Codesniffer spotted this on the [gallery page](https://viktorkoves.com/gallery), where it pointed out that none of the images have alt tags :grimacing: 

This should be remedied on the gallery and portfolio pages. The about page's one image already has an `alt` tag.
